cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,151 @@
+from django.test import TestCase
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+
+
+class IndexPageTestCase(WagtailTestUtils, TestCase):
+    """Test IndexPage model properties and methods."""
+
+    def setUp(self):
+        self.page = IndexPageFactory(
+            title="Test Index Page",
+            description="This is an example",
+            content="This is the main content",
+            featured_pages=None,
+            related_links=None,
+        )
+
+        self.page_url = self.page.url
+
+    def test_no_featured_pages_displayed_when_no_children_and_no_custom_featured_pages_selected(self):
+        """Test that the Featured Items block isn't displayed when the Index Page has no child pages
+        and no custom Featured Items are specified.
+        """
+        self.page.featured_pages = None
+
+        response = self.client.get(self.page_url)
+        self.page.save_revision().publish()
+
+        self.assertEqual(response.status_code, 200)
+
+        self.assertNotContains(response, "ons-document-list")
+
+    def test_children_displayed_as_featured_pages_when_no_custom_featured_pages(self):
+        """Test that the children pages of the Index Page are displayed
+        when no custom Featured Items are specified.
+        """
+        child_page = InformationPageFactory(parent=self.page)
+
+        self.page.save_revision().publish()
+
+        response = self.client.get(self.page_url)
+
+        self.assertEqual(response.status_code, 200)
+
+        featured_page_html = f"""
```

Comment:
this is brittle in that any changes to the DS component will result in the tests failing. I would rather test that the links are there in the content

Reply:
I'm going to test that all the 3 keys (`title`, `url` and `description`) that are returned from `get_formatted_items()` are on the page, because what this test is effectively doing is testing the funciton. Lmk what you think.

Reply:
[Improved tests](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/ce09d7c32fb17fa2d359b682c30e5c92bbdf0ef1)